dfget/core/downloader/p2p_downloader/client_writer.go

```diff
@@ -196,12 +197,18 @@ func startSyncWriter(queue util.Queue) util.Queue {
 }
 
 func (cw *ClientWriter) sendSuccessPiece(piece *Piece, cost time.Duration) {
-	cw.api.ReportPiece(piece.SuperNode, &types.ReportPieceRequest{
+	resp, e := cw.api.ReportPiece(piece.SuperNode, &types.ReportPieceRequest{
 		TaskID:     piece.TaskID,
 		Cid:        cw.cfg.RV.Cid,
 		DstCid:     piece.DstCid,
 		PieceRange: piece.Range,
 	})
+	if e != nil {
+		logrus.Errorf("sendSuccessPiece error:%v", e)
```

Comment:
`logrus.Errorf("failed to report piece: %v", err)`
Please see https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule005---generation-of-action-failure

In addition, you should add a white space between `:` and the detailed error message.

Reply:
Done